src/getHumanAge.test.js

```diff
@@ -7,4 +7,54 @@ describe('getHumanAge', () => {
     expect(getHumanAge)
       .toBeInstanceOf(Function);
   });
+
+  test('should return [0, 0] if both pets are less than 15 year old', () => {
+    expect(getHumanAge(14, 14)).toEqual([0, 0]);
+  });
+
+  test('should return [0, 0] if both pets are 0 years old', () => {
+    expect(getHumanAge(0, 0)).toEqual([0, 0]);
+  });
+
+  test(`should return [1, 1] if both pets are 15 years old
+    and less then 24`, () => {
+    expect(getHumanAge(15, 23)).toEqual([1, 1]);
+  });
+
+  test('should return [2, 2] if both pets are exactly 24 years old', () => {
+    expect(getHumanAge(24, 24)).toEqual([2, 2]);
+  });
+
+  test(`should return [2, 2] if both pets are
+    between 24 and 27 years old`, () => {
+    expect(getHumanAge(24, 27)).toEqual([2, 2]);
+  });
+
+  test('should return [3, 2] if both pets are 28 years old', () => {
+    expect(getHumanAge(28, 28)).toEqual([3, 2]);
+  });
```

Comment:
The description of this test case says 'should return [3, 2] if both pets are 28 years old', but the expected output is [3, 3]. Please ensure that the test description matches the expected output.